flang/test/Lower/OpenACC/acc-data.f90

```diff
@@ -0,0 +1,69 @@
+! This test checks lowering of OpenACC data directive.
+
+! RUN: bbc -fopenacc -emit-fir %s -o - | FileCheck %s
+
+program acc_data
+  real, dimension(10, 10) :: a, b, c
+
+!CHECK: [[A:%.*]] = fir.alloca !fir.array<10x10xf32> {name = "a"}
```

Comment:
I think most of our tests leave the `%` out of the regexp as an anchor for the match.

Reply:
Right. I'll update that.

Reply:
I checked in the mlir directory and when the value is captured the `%` is kept in the regex. So Ieft it as well. 

https://github.com/llvm/llvm-project/blob/61ac58e10ae1fbb18416cac20119725803b9ad6f/mlir/test/Conversion/SCFToStandard/convert-to-cfg.mlir#L229